uitools/import/Esri/ArcGISRuntime/Toolkit/Controller/FloorFilterController.qml

```diff
@@ -34,22 +110,42 @@ QtObject {
         return null
     }
 
-    property int updateLevelsMode: FloorFilterController.UpdateLevelsMode.AllLevelsMatchingVerticalOrder
+    property int updateLevelsMode: FloorFilterController.UpdateLevelsMode.SingleLevel
 
     property bool autoselectSingleFacilitySite
 
+    property var automaticSelectionMode
+
     property FloorManager floorManager
 
     //refresh()?
     property string selectedLevelId
 
     property string selectedFacilityId
 
+
+    /*!
+      if levelId == "", disable visibility previous level
+      */
+    function setSelectedLevelId(levelId) {
+        let idx = findElementIdxById(levelId,
+                                     FloorFilterController.TypeElement.Level)
+        if (levelId === "" && idx != null) {
+            // reset the previous level visibility
+            if (internal.selectedLevel) {
+                internal.selectedLevel.visible = false
+            }
```

Comment:
the visiblity disable might be extracted in a separate function.
But if we agree that passing an empty string means to disable visibility of current level, can remain as it is.

Reply:
@anmacdonald what do you think about this? Is it ok to integrate the `disableVisiblityLastLevel` into the `setSelectedLevelId` if passing an empty string levelId?
I dont think there can be Ids with empty strings, so should be safe